osquery/tables/CMakeLists.txt

```diff
@@ -22,6 +22,7 @@ if(APPLE)
   ADD_OSQUERY_LINK_ADDITIONAL("libxar.dylib")
   ADD_OSQUERY_LINK_ADDITIONAL("magic")
   ADD_OSQUERY_LINK_ADDITIONAL("tsk")
+  ADD_OSQUERY_LINK_ADDITIONAL("lldpctl")
```

Comment:
Well, in the current if/else block its in, yes it does, because it's in there once for Apple, and once for Linux (not Apple, Windows, or FreeBSD to be exact).  But there is another if block below with `(APPLE OR LINUX)`, although most of the dependencies seem to be in the previous if/else block.   So should I be putting in the [latter](https://github.com/digitalocean/osquery/blob/c73c7bc5ca898d4dec47317ac4d5a666f6862df4/osquery/tables/CMakeLists.txt#L91) block?